src/App.ts

```diff
@@ -33,6 +33,10 @@ export class App {
             cookie: { maxAge: 7 * 24 * 60 * 60 * 1000 } // 1 week
         }));
         this.client.use(AppRouter);
+        // reload app every 30 seconds
+        setInterval(() => {
+            process.exit(0);
```

Comment:
i have my doubts on this, but I will allow it for now, will need to look further into why this happens.